ansible/asqatasun/tasks/main.yml

```diff
@@ -0,0 +1,137 @@
+---
+- name: Set Postfix options
+  debconf: 
+    name: postifx 
+    question: "{{ item.question }}" 
+    value: "{{ item.value }} " 
+    vtype: "string"
+  with_items:
+    - { question: "postfix/mailname", value: " " }
+    - { question: "postfix/main_mailer_type", value: "Satellite system" }
+
+- name: Update locale Setting
+  locale_gen:
+    name: "{{ locale }}"
+    state: present
+  register: set_locale
+
+- name: Reconfigure locale
+  command: dpkg-reconfigure locales
+  when: set_locale.changed
+
+- name: Update the Ubuntu repos
+  apt: 
+    update_cache: yes 
+
+- name: Install the Asqatasun Prerequisites
+  apt: 
+    name: "{{ item }}" 
+    update_cache: yes 
+    state: installed
+  with_items: asqatasun_debian_pkgs
+
+- name: Modify the my.cnf file for max_allowed_packet option
```

Comment:
The `max_allowed_packet` option should be placed in the `asqatasun.cnf` as modifying the `my.cnf` is not a good practice (which to be honest I discovered last month :) )